ObstacleClassifier:
  map_frame: map
  robot_base_radius: 0.2 # must be the same with the one in costmap_common_params.yaml